samples/Uno.Toolkit.Samples/Uno.Toolkit.Samples.Shared/Content/Controls/DividerSamplePage.xaml

```diff
@@ -133,6 +133,77 @@
 					</StackPanel>
 				</DataTemplate>
 			</sample:SamplePageLayout.MaterialTemplate>
+			<sample:SamplePageLayout.M3MaterialTemplate>
+				<DataTemplate>
+					<StackPanel Padding="0,8">
+
+						<StackPanel Background="{StaticResource MaterialSurfaceBrush}"
+									Margin="8">
+							<ContentControl ContentTemplate="{StaticResource DefaultItemTemplate}" />
+
+							<utu:Divider Style="{StaticResource M3MaterialDividerStyle}" />
+
+							<ContentControl ContentTemplate="{StaticResource DefaultItemTemplate}" />
+						</StackPanel>
+
+						<StackPanel Background="{StaticResource MaterialSurfaceBrush}"
+									Margin="8">
+
+							<ContentControl ContentTemplate="{StaticResource DefaultItemTemplate}" />
+
+							<utu:Divider Style="{StaticResource M3MaterialDividerStyle}"
+										 SubHeader="SubHeader" />
+
+							<ContentControl ContentTemplate="{StaticResource DefaultItemTemplate}" />
+						</StackPanel>
+
+						<StackPanel Background="{StaticResource MaterialPrimaryBrush}"
+									Margin="8">
+
+							<ContentControl ContentTemplate="{StaticResource CustomItemTemplate}" />
+
+							<utu:Divider Style="{StaticResource M3MaterialDividerStyle}"
+										 Foreground="{StaticResource MaterialOnPrimaryLowBrush}" />
```

Comment:
we can ignore about resources used in elsewhere. but for the showcased control itself, it is best to use equivalent/similar resources in m3.